plugin/src/test/java/com/epam/grandhackathon/deployment/sphere/plugin/metadata/persistence/dao/BuildMetaDataDaoMock.java

```diff
@@ -0,0 +1,21 @@
+package com.epam.grandhackathon.deployment.sphere.plugin.metadata.persistence.dao;
+
+import java.util.Collection;
+
+import com.epam.grandhackathon.deployment.sphere.plugin.metadata.model.BuildMetaData;
+
+public class BuildMetaDataDaoMock extends BuildMetaDataDao {
```

Comment:
For guice injection:
Injector injector = Guice.createInjector(new AbstractModule() {
            @Override
            protected void configure() {
            	bind(BuildMetaDataDao.class).to(BuildMetaDataDaoMock.class);
            }
        });
With mock it didn't work.